tests/fread/test-fread-small.py

```diff
@@ -1207,3 +1207,11 @@ def test_qr_bump_out_of_sample(capsys):
     pysrc[0][137] = -1
     pysrc[1][137] = '"This" is not funny'
     assert d0.to_list() == pysrc
+
+def test_sep_within_quoted_fields():
+    src1 = ['"abc"\n"1,000"\n"2,000"\n']
+    src2 = ['movies\n"encore, once more encore!"\n"great gatsby"']
```

Comment:
`src1` and `src2` should be separated into 2 distinct tests, because they are unrelated to each other (and therefore can fail or succeed independently from each other).

Also, they don't have to be lists - plain strings will do.

Reply:
Done